core/templates/pages/topic-viewer-page/practice-tab/practice-tab.component.html

```diff
@@ -1,35 +1,55 @@
                         (change)="checkIfQuestionsExist(selectedSubtopicIndices)">
             <span [ngClass]="{'subtopic-title': displayArea === 'progressTab'}">
               {{ subtopic.getTitle() }}
             </span>
           </mat-checkbox>
-          <div class="capsule-progress" *ngIf="displayArea === 'progressTab'" [ngStyle]="{'background': 'linear-gradient(to right, #205D86 ' + getBackgroundForProgress(i) + '%,white ' + getBackgroundForProgress(i) + '%)'}">
-            <span class="subtopic-mastery" [ngStyle]="{'right': subtopicMasteryPosition(i) + 'px', 'color': masteryTextColor(i)}">
+          <div class="capsule-progress"
+               *ngIf="displayArea === 'progressTab'"
+               [ngStyle]="{'background': 'linear-gradient(to right, #205D86 ' + getBackgroundForProgress(i) + '%,white ' + getBackgroundForProgress(i) + '%)'}">
+            <span class="subtopic-mastery"
+                  [ngStyle]="{'right': subtopicMasteryPosition(i) + 'px', 'color': masteryTextColor(i)}">
               {{subtopicMasteryArray[i]}}%
             </span>
           </div>
         </div>
       </mat-card-content>
       <mat-card-actions class="card-actions">
         <div class="fx-row fx-main-center fx-cross-center full-width">
-          <button mat-raised-button [disabled]="isStartButtonDisabled()" (click)="openNewPracticeSession()" class="practice-session-button protractor-test-practice-start-button">
-            <span *ngIf="questionsStatusCallIsComplete" class="font-heavy" [innerHTML]="'I18N_TOPIC_VIEWER_START_PRACTICE' | translate"></span>
-            <mat-spinner [diameter]="20" class="question-status-spinner" *ngIf="!questionsStatusCallIsComplete"></mat-spinner>
+          <button mat-raised-button
+                  [disabled]="isStartButtonDisabled() && questionsStatusCallIsComplete"
+                  [ngClass]="{'practice-button-background-status-pending': !questionsStatusCallIsComplete, 'practice-button-background-status-complete': questionsStatusCallIsComplete}"
+                  (click)="openNewPracticeSession()"
+                  class="practice-session-button protractor-test-practice-start-button" >
+            <span *ngIf="questionsStatusCallIsComplete"
+                  class="font-heavy"
+                  [innerHTML]="'I18N_TOPIC_VIEWER_START_PRACTICE' | translate">
+            </span>
+            <mat-spinner [diameter]="20"
+                         class="question-status-spinner"
+                         *ngIf="!questionsStatusCallIsComplete">
+            </mat-spinner>
           </button>
         </div>
-        <div class="mt-3" *ngIf="questionsStatusCallIsComplete && !questionsAreAvailable && isAtLeastOneSubtopicSelected()">
+        <div class="mt-3"
+             *ngIf="questionsStatusCallIsComplete && !questionsAreAvailable && isAtLeastOneSubtopicSelected()">
           <em class="text-danger">
             There are no questions created yet for the selected subtopic(s).
```

Comment:
I know this is out of scope for this PR, but if possible, can you add a translation key for this? (Fine, if not, just mentioning it here as I just noticed it)

Reply:
yah! i'll do it!

Reply:
Done!